obelisk_ws/src/robots/achilles_model/mujoco/achilles.xml

```diff
@@ -0,0 +1,303 @@
+            </body>
+          </body>
+        </body>
+
+      </body>
+
+   </worldbody>
+
+  <actuator>
+
+    <motor name="left_hip_yaw_tor"  joint="left_hip_yaw_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="left_hip_roll_tor"  joint="left_hip_roll_joint" ctrllimited="true" ctrlrange="-70  70" />
+    <motor name="left_hip_pitch_tor"  joint="left_hip_pitch_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="left_knee_pitch_tor"  joint="left_knee_pitch_joint" ctrllimited="true" ctrlrange="-70  70" />
+    <motor name="left_ankle_pitch_tor"  joint="left_ankle_pitch_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="left_shoulder_pitch_tor"  joint="left_shoulder_pitch_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="left_shoulder_roll_tor"  joint="left_shoulder_roll_joint" ctrllimited="true" ctrlrange="-12  12" />
+    <motor name="left_shoulder_yaw_tor"  joint="left_shoulder_yaw_joint" ctrllimited="true" ctrlrange="-12  12" />
+    <motor name="left_elbow_pitch_tor"  joint="left_elbow_pitch_joint" ctrllimited="true" ctrlrange="-24  24" />
+
+    <motor name="right_hip_yaw_tor"  joint="right_hip_yaw_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="right_hip_roll_tor"  joint="right_hip_roll_joint" ctrllimited="true" ctrlrange="-70  70" />
+    <motor name="right_hip_pitch_tor"  joint="right_hip_pitch_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="right_knee_pitch_tor"  joint="right_knee_pitch_joint" ctrllimited="true" ctrlrange="-70  70" />
+    <motor name="right_ankle_pitch_tor"  joint="right_ankle_pitch_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="right_shoulder_pitch_tor"  joint="right_shoulder_pitch_joint" ctrllimited="true" ctrlrange="-35 35" />
+    <motor name="right_shoulder_roll_tor"  joint="right_shoulder_roll_joint" ctrllimited="true" ctrlrange="-12 12" />
+    <motor name="right_shoulder_yaw_tor"  joint="right_shoulder_yaw_joint" ctrllimited="true" ctrlrange="-12 12" />
+    <motor name="right_elbow_pitch_tor"  joint="right_elbow_pitch_joint" ctrllimited="true" ctrlrange="-24  24" />
+
+  </actuator>
+
+  <contact>
+    <!--pair name="left_foot_to_plane" geom1="left_foot" geom2="plane" friction="1 1 0.001 0.001 0.001"/>
+    <pair name="right_foot_to_plane" geom1="right_foot" geom2="plane" friction="1 1 0.001 0.001 0.001"/-->
+
+    <pair name="torso_to_plane" geom1="torso_collision_box" geom2="plane" friction="1 1 0.001 0.001 0.001"/>
+
+    <pair name="foot_front_left_to_plane" geom1="foot_front_left" geom2="plane" friction="1 1 0.001 0.001 0.001"/>
+    <pair name="foot_rear_left_to_plane" geom1="foot_rear_left" geom2="plane" friction="1 1 0.001 0.001 0.001"/>
+    <pair name="foot_front_right_to_plane" geom1="foot_front_right" geom2="plane" friction="1 1 0.001 0.001 0.001"/>
+    <pair name="foot_rear_right_to_plane" geom1="foot_rear_right" geom2="plane" friction="1 1 0.001 0.001 0.001"/>
```

Comment:
i would add comments explaining the reasoning for some of these settings for posterity